src/cmdc_tools/datasets/official/DE/data.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd
+import requests
+import us
+
+from ...base import DatasetBaseNoDate
+from ..base import CountyData
+
+
+class Delaware(DatasetBaseNoDate, CountyData):
+    state_fips = int(us.states.lookup("Delaware").fips)
+    has_fips: bool = False
+    source_map = {
+        "Sussex": "https://myhealthycommunity.dhss.delaware.gov/locations/county-sussex/download_covid_19_data",
+        "Kent": "https://myhealthycommunity.dhss.delaware.gov/locations/county-kent/download_covid_19_data",
+        "New Castle": "https://myhealthycommunity.dhss.delaware.gov/locations/county-new-castle/download_covid_19_data",
+    }
+    source = list(source_map.values())
+
+    def get(self):
+        dfs = []
+        for (county, url) in self.source_map.items():
+            df = self._get_from_source(url)
+            df["county"] = county
+            dfs.append(df)
+
+        return pd.concat(dfs, sort=False).melt(
+            id_vars=["dt", "county"], var_name="variable_name"
+        )
+
+    def _get_from_source(self, source):
+        df = pd.read_csv(source)
+
+        df["dt"] = pd.to_datetime(df[["Year", "Month", "Day"]])
+        keep = df.loc[df.Unit == "people"][
+            ["Location", "County", "dt", "Statistic", "Value"]
+        ]
+        pivoted = (
```

Comment:
Does this need to be pivoted or could you use `df["Statistic"].replace(...)` and then change the name of the `Statistic` column to `variable_name`?

Reply:
Maybe...haven't done that before. I thought it would be easier to manipulate the columns if I pivoted it